scripts/tests/test_user_system_tag_normalization.py

```diff
@@ -0,0 +1,181 @@
+import pytest
+import pytz
+from datetime import datetime
+
+from osf_tests.factories import PreprintFactory, UserFactory, ProjectFactory, TagFactory
+from osf.models import Tag
+from scripts.normalize_user_tags import normalize_provider_source_tags, normalize_campaign_source_tags, add_provider_claimed_tags, add_campaign_claimed_tags, add_osf_provider_tags, add_prereg_campaign_tags, PROVIDER_SOURCE_TAGS
+
+pytestmark = pytest.mark.django_db
+
+LEGACY_PROVIDER_SOURCE_TAG_NAMES = [
+    'africarxiv_preprints',
+    'agrixiv_preprints',
+    'arabixiv_preprints',
+    'bitss_preprints',
+    'eartharxiv_preprints',
+    'ecoevorxiv_preprints',
+    'ecsarxiv_preprints',
+    'engrxiv_preprints',
+    'focusarchive_preprints',
+    'frenxiv_preprints',
+    'inarxiv_preprints',
+    'lawarxiv_preprints',
+    'lissa_preprints',
+    'marxiv_preprints',
+    'mediarxiv_preprints',
+    'mindrxiv_preprints',
+    'nutrixiv_preprints',
+    'osf_preprints',
+    'paleorxiv_preprints',
+    'psyarxiv_preprints',
+    'socarxiv_preprints',
+    'sportrxiv_preprints',
+    'thesiscommons_preprints',
+    'bodoarxiv_preprints',
+    'osf_registries',
+]
+
+NORMALIZED_PROVIDER_SOURCE_TAG_NAMES = [
+    'source:provider|preprint|africarxiv',
+    'source:provider|preprint|agrixiv',
+    'source:provider|preprint|arabixiv',
+    'source:provider|preprint|metaarxiv',
+    'source:provider|preprint|eartharxiv',
+    'source:provider|preprint|ecoevorxiv',
+    'source:provider|preprint|ecsarxiv',
+    'source:provider|preprint|engrxiv',
+    'source:provider|preprint|focusarchive',
+    'source:provider|preprint|frenxiv',
+    'source:provider|preprint|inarxiv',
+    'source:provider|preprint|lawarxiv',
+    'source:provider|preprint|lissa',
+    'source:provider|preprint|marxiv',
+    'source:provider|preprint|mediarxiv',
+    'source:provider|preprint|mindrxiv',
+    'source:provider|preprint|nutrixiv',
+    'source:provider|preprint|osf',
+    'source:provider|preprint|paleorxiv',
+    'source:provider|preprint|psyarxiv',
+    'source:provider|preprint|socarxiv',
+    'source:provider|preprint|sportrxiv',
+    'source:provider|preprint|thesiscommons',
+    'source:provider|preprint|bodoarxiv',
+    'source:provider|registry|osf',
+]
+
+PROVIDER_CLAIMED_TAG_NAMES = [
```

Comment:
Could you import https://github.com/CenterForOpenScience/osf.io/pull/9022/files#diff-e454a52b84234680cffb5f12a0695a05R49 instead of repeating? (Plus a similar question for any of these others lists)